components/footer/Footer.js

```diff
@@ -26,13 +27,22 @@ const Footer = ({ children }) => {
         fluid
         className="position-fixed bottom-0 bg-primary text-white-50"
       >
-        <Container className="h-100">
-          <HelpChat
-            helpChatText="How can I help you?"
-            helpChatTextSmallScreen="Help?"
-          >
-            {children}
-          </HelpChat>
+        <Container className="h-100 d-flex align-items-center justify-content-between">
+          <div>
+            <HelpChat
+              helpChatText="How can I help you?"
+              helpChatTextSmallScreen="Help?"
+            >
+              {children}
+            </HelpChat>
+          </div>
+          <div>
+            <Navigation
```

Comment:
The `Navigation` buttons are visible on the `welcome` page - we only want to render the `Navigation` component on pages past the `welcome` page, as the "Get Started" button takes place of the nav in `welcome`. 

let's refactor the `Footer` component to take a prop to render the `Navigation` bar conditionally, then update `levels` page and `welcome` page accordingly so that `welcome` does not render the nav in the footer.

Reply:
Done